src/index.html

```diff
@@ -1,14 +1,21 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+        content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <div class="container">
+      <div class="container_clock">
+        <div class="container_string-first"></div>
+        <div class="container_string-second"></div>
+        <div class="container_circle"></div>
+      </div>
```

Comment:
Please follow BEM and give youre classes more representive names
Example:
```suggestion
      <div class="clock">
        <div class="clock__minutes-arrow"></div>
        <div class="clock__seconds-arrow"></div>
        <div class="clock__central-dot"></div>
      </div>
```
This way it's easy to understand which element does what